cabal.project

```diff
@@ -15,3 +15,14 @@ source-repository-package
     location: https://github.com/plow-technologies/hspec-golden-cereal.git
     tag: f9e3e485409c5a1de1de99a8ec0f35226c79da79
     --sha256: j+SZk5AZsNP674fb+1aiA7vrsk6Eq5BQM2losQSnaeE=
+
+-- NOTE
+-- Do not remove or change the following comment! It's necessary to make
```

Comment:
Yes, it is evil. But I couldn't find another solution. We were thinking about splitting `inferno-ml` into its own repo, which would avoid this. But we wanted to get some reviews in for everything instead of just creating a new repo with all of the changes

Reply:
It's a good solution given the circumstances